components/scream/src/share/atm_process/atmosphere_process.cpp

```diff
@@ -828,21 +826,35 @@ ::remove_group (const std::string& group_name, const std::string& grid_name) {
 
 void AtmosphereProcess::setup_conservation_checks_for_this_process (const int dt)
 {
-  const Real tol = m_params.get<Real>("conservation check tolerance",
-                                      std::numeric_limits<Real>::max());
+  // Only setup the check if this process call it.
+  if (not m_params.get<bool>("enable_column_conservation_checks", false)) {
```

Comment:
Since this is being done at every process call, I recommend calling the PL get function once at init and saving the value to a class boolean that is then used during time stepping.